### PR TITLE
frontend: hidden wallet error messages

### PR DESCRIPTION
**Pull request by benma**
Was cryptic coming from backend.

### CODE REVIEW

**Comment by benma** [comment]
@TheCharlatan please check and approve :)

File: frontends/web/src/routes/device/settings/components/hiddenwallet.jsx [hunk 1]
```diff
@@ -127,6 +129,7 @@ export default class HiddenWallet extends Component {
                                     onValidPassword={this.setValidPIN} />
                                 <PasswordRepeatInput
                                     idPrefix="password"
+                                    pattern="^.{8,}$"
```
Comment (thisconnect) [inline frontends/web/src/routes/device/settings/components/hiddenwallet.jsx]: The pattern attribute is nice, but if it is always active you can't submit the form in this case you never see the error. A user would not be informed that the password is too short when the input field only gets red.

Comment (benma) [reply frontends/web/src/routes/device/settings/components/hiddenwallet.jsx]: Makes sense. Should I remove the pattern, or add a small text `min. 8 characters`?

Comment (thisconnect) [reply frontends/web/src/routes/device/settings/components/hiddenwallet.jsx]: yes please remove the attribute for now

Comment (thisconnect) [reply frontends/web/src/routes/device/settings/components/hiddenwallet.jsx]: We could add it later in another PR, after we are sure there was an error, but maybe have to allow submitting even if it is invalid.

**Comment by benma** [comment]
@thisconnect done

**Comment by thisconnect** [comment]
thanks

**Pull request merged**